pages/index.tsx

```diff
@@ -16,54 +17,25 @@ import LoginWithEmailWebAuthn from '../components/EmailWebAuthn/LoginWithEmail';
 //
 // VERCEL_URL only contains the domain of the site's URL, the scheme is not included so we must add it manually,
 // see https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables.
```

Comment:
Please move this comment into `lib/getRedirectBaseUrl.ts`

Reply:
fixed https://github.com/stytchauth/stytch-nextjs-integration/pull/47/commits/e84494220b585655fc9847e0439b2aa8c9109d05